jumanji/environments/routing/op/env.py

```diff
@@ -0,0 +1,313 @@
+from typing import Optional, Sequence, Tuple
+
+import chex
+import jax
+import jax.numpy as jnp
+import matplotlib
+from chex import PRNGKey
+from numpy.typing import NDArray
+
+
+import jumanji
+from jumanji import specs
+from jumanji.env import Environment
+from jumanji.types import TimeStep, restart, termination, transition
+from jumanji.viewer import Viewer
+import generator, types_, reward, constants
+
+
+class OP(Environment[types_.State]):
```

Comment:
Please try to match the style of other environments. For instance, it seems that other env import State from types, hence then just call State rather than types.State.